Content.Shared/GameObjects/Components/Body/Part/SharedBodyPartComponent.cs

```diff
@@ -313,6 +314,15 @@ private void RemovedFromBody(IBody old)
         protected virtual void OnAddedToBody(IBody body) { }
 
         protected virtual void OnRemovedFromBody(IBody old) { }
+
+        public void Gib()
+        {
+            foreach (var mechanism in _mechanisms)
+            {
+                RemoveMechanism(mechanism);
+                mechanism.Owner.RandomOffset(0.25f);
```

Comment:
Same misprediction problem here when called from the client.

Reply:
Done